ECOSYS-101 Payara VSCode Extension (#721)
payara/Payara-Server-Documentation: documentation/ecosystem/vscode-extension.adoc

= Payara VSCode Plugin

Payara Platform provides the Payara extension to integrate Payara Server into the Visual Studio Code.

> The plugins are installable from the link:https://marketplace.visualstudio.com/items?itemName=Payara.payara-vscode[Visual Studio Marketplace].

Reviewer: ```suggestion
The extension is available in the link:https://marketplace.visualstudio.com/items?itemName=Payara.payara-vscode[Visual Studio Marketplace].
```
